docs/customization.md

```diff
@@ -0,0 +1,93 @@
+---
+id: customization
+title: Customization
+---
+## Customisation
+You may customise darkroom, for example, you may want to write a storage backend that talks to another service and gets the images.
```

Comment:
Change this and all other occurrences from `customise` to `customize` just to be consistent.